src/main/java/com/simprints/libsimprints/Identification.kt

```diff
@@ -24,4 +25,28 @@ data class Identification(
         confidence < other.confidence -> 1
         else -> -1
     }
+
+    override fun describeContents(): Int = 0
+
+    override fun writeToParcel(dest: Parcel, flags: Int) {
+        dest.writeString(guid)
+        dest.writeInt(this.confidence)
+        dest.writeInt(this.tier.ordinal)
+    }
+
+    companion object {
+        @JvmField
+        val CREATOR = object : Creator<Identification> {
+
+            override fun createFromParcel(parcel: Parcel): Identification? {
+                val guid = parcel.readString().orEmpty()
```

Comment:
Have you confirmed that `createFromParcel` is working as expected? I remember having fun with it back in the days

Reply:
As per slack conversation - I have integrated this version in SID and then tried calling from the current "main" version of Intent Launcher (it still uses java version of LibSimprints). So I am fairly confident that it works correctly.